src/Components/WebAssembly/WebAssembly/src/Microsoft.AspNetCore.Components.WebAssembly.csproj

```diff
@@ -31,6 +25,11 @@
     <Compile Include="$(ComponentsSharedSourceRoot)src\JsonSerializerOptionsProvider.cs" />
     <Compile Include="$(ComponentsSharedSourceRoot)src\WebEventData.cs" />
     <Compile Include="$(ComponentsSharedSourceRoot)src\ElementReferenceJsonConverter.cs" />
+    <Compile Include="$(ComponentsSharedSourceRoot)src\ComponentParametersTypeCache.cs" />
+    <Compile Include="$(ComponentsSharedSourceRoot)src\RootComponentTypeCache.cs" />    
+    <Compile Include="$(SharedSourceRoot)Components\ClientComponentSerializationSettings.cs" Link="Prerendering/ClientComponentSerializationSettings.cs" />
+    <Compile Include="$(SharedSourceRoot)Components\ClientComponentMarker.cs" Link="Prerendering/ClientComponentMarker.cs" />
```

Comment:
Are `ClientComponentSerializationSettings` and `ClientComponentMarker` used for both server-side components and webassembly ones? It seems like probably so since they are now in shared source, but I can't see any corresponding older server-specific versions of the code having been removed in favour of the new shared ones.

Or if these are wasm-specific, would it be possible rename them to `WebAssemblyComponentSerializationSettings` and `WebAssemblyComponentMarker`? We've been preferring webassembly as the general term rather than "client" due to it being less ambiguous.

Reply:
I've renamed them. 

I'm keeping the markers separate between SSB and Wasm. While they share some things in common, they represent implementation details so it's better to keep them separate to avoid accidental changes.